src/matlab2tikz.m

```diff
@@ -1502,6 +1502,16 @@ case guitypes()
 
     options = setAxisTicks(m2t, options, axis, pgfTicks, pgfTickLabels, ...
         hasMinorTicks, tickDirection, isDatetimeTicks);
+        
+    options = setAxisTickLabelStyle(m2t, options,axis,handle);
+end
+% ==============================================================================
+function options = setAxisTickLabelStyle(m2t, options, axis, handle)
+    keywordTickLabelRotation = [upper(axis), 'TickLabelRotation'];
+    Rotation = getOrDefault(handle, keywordTickLabelRotation, 0);
```

Comment:
`keyword` sounds like a good compromise to me. Indeed, the name was somewhat on the long side. :+1: